openlibrary/plugins/openlibrary/js/type_changer.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Functionality for TypeChanger.html
+ */
+
+export function initTypeChanger() {
+    // /about?m=edit - where this code is run
+
+    function changeTemplate() {
+        // Change the template of the page based on the selected value
+        const searchParams = new URLSearchParams(window.location.search);
+        const t = document.getElementById('type.key').value;
+        searchParams.set('t', t);
+        window.location.search = searchParams.toString();
+    }
+    document.getElementById('type.key').onchange = changeTemplate;
```

Comment:
```suggestion
    elem.addEventListener('change', () => changeTemplate())
```
Use `addEventListener` instead of setting `onchagne` directly, here.